src/main/resources/com/google/api/codegen/java/stub.snip

```diff
@@ -58,6 +62,79 @@
           .build();
 @end
 
+@private httpMethodDescriptor(methodDescriptor)
```

Comment:
nit: I'd lean towards s/`http`/`rest`/ but if you think this is better for consistency with existing internal code, that's fine.

Reply:
Yes, this is for consistency and compatibility with discogapic. I can't make big changes here untill discogapic is removed.

Reply:
ok